libs/MVS/SceneTexture.cpp

```diff
@@ -509,11 +517,34 @@ bool MeshTexture::ListCameraFaces(FaceDataViewArr& facesDatas, float fOutlierThr
 		faceMap.create(imageData.height, imageData.width);
 		depthMap.create(imageData.height, imageData.width);
 		RasterMesh rasterer(vertices, imageData.camera, depthMap, faceMap);
+		// creating mask for the image border
+		rasterer.mask = Image8U(imageData.height + 2, imageData.width + 2);
+		rasterer.mask = rasterer.mask != rasterer.mask;
+		Image8U imageCopy;
+		cv::cvtColor(imageData.image, imageCopy, cv::COLOR_BGR2GRAY);
+		imageCopy -= 1; // this line gets rid of white spots in the middle of the image
+		cv::floodFill(imageCopy, rasterer.mask, cv::Point(0, 0), 255);
+		cv::floodFill(imageCopy, rasterer.mask, cv::Point(0, imageCopy.rows / 2), 255);
+		cv::floodFill(imageCopy, rasterer.mask, cv::Point(0, imageCopy.rows - 1), 255);
+		cv::floodFill(imageCopy, rasterer.mask, cv::Point(imageCopy.cols / 2, imageCopy.rows - 1), 255);
+		cv::floodFill(imageCopy, rasterer.mask, cv::Point(imageCopy.cols - 1, imageCopy.rows - 1), 255);
+		cv::floodFill(imageCopy, rasterer.mask, cv::Point(imageCopy.cols - 1, imageCopy.rows / 2), 255);
+		cv::floodFill(imageCopy, rasterer.mask, cv::Point(imageCopy.cols - 1, 0), 255);
+		cv::floodFill(imageCopy, rasterer.mask, cv::Point(imageCopy.cols / 2, 0), 255);
+		cv::Mat kernel = cv::getStructuringElement(cv::MORPH_ELLIPSE, cv::Size(3, 3));
+		cv::erode(rasterer.mask, rasterer.mask, kernel);
+		rasterer.mask = rasterer.mask == 0;
+		if (VERBOSITY_LEVEL > 2) {
+			cv::imwrite(String::FormatString("invalidMask%d.png", idx), rasterer.mask);
```

Comment:
pls format the index with 4 numbers, with leading 0s
this is required in order to correctly list the files in incremental order